my_list.rb

```diff
@@ -0,0 +1,13 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerab
```

Comment:
- [ ] We appear to have an unitialized constant `MyEnumerab` here as shown below 👇. This is as a result of a typo on your end: it should be `include MyEnumerable` not `include MyEnumerab`. Please fix this bug 🐛 
![image](https://github.com/kamranasif506/Enumerable/assets/46839548/da6acf7b-7128-40e8-b08e-ed3e7bdb642f)